functions-methods.js

```diff
@@ -33,4 +72,35 @@
 // checkEmailValidity("tessmellink@novi.nl") geeft true - want @ en punt op de juiste plek
 // checkEmailValidity("n.eekenanovi.nl") geeft false - want geen @
 // checkEmailValidity("n.eeken@novinl.") geeft false - want de punt mag niet als laatst
-// checkEmailValidity("tessmellink@novi,nl") geeft false - want er staat een komma in
\ No newline at end of file
+// checkEmailValidity("tessmellink@novi,nl") geeft false - want er staat een komma in
+
+//Definieer functie
+//Voeg parameter email toe
+//Controleer of de waarde van het meegegeven email adres een @ bevat
+//Controleer of de waarde van het meegegeven email adres geen , bevat
+//Controleer of er geen . als allerlaatste karakter in voorkomt
+//Schrijf een if statement om te kijken of dit waar of niet waar is
+//Geef de uitkomst terug
+//Roep de functie aan en geef argument mee
+//Log de uitkomst in de console
+
+const checkEmailValidity = (email) => {
+    const hasAtInEmail = email.includes("@");
+    const hasCommaInEmail = email.includes(",");
+    const lastCharEmailIsDot = email.charAt(email.length -1) === ".";
+
+    if (hasAtInEmail && !hasCommaInEmail && !lastCharEmailIsDot) {
+        return true + " - want @ en punt op de juiste plek";
+    } else if (!hasAtInEmail) {
+        return false + " - want geen @";
+    } else if (hasCommaInEmail) {
+        return false + " - geeft false want er staat een komma in";
+    } else if (lastCharEmailIsDot) {
+        return false + " - geeft false want de punt mag niet als laatst";
+    }
```

Comment:
De eerste check is een inverse van alles wat erna komt. Ik zou het op deze manier schrijven:
```
if (!hasAtInEmail) {
    return false + " - want geen @";
} else if (hasCommaInEmail) {
    return false + " - geeft false want er staat een komma in";
} else if (lastCharEmailIsDot) {
    return false + " - geeft false want de punt mag niet als laatst";
}

return true + " - want @ en punt op de juiste plek";
```